crates/bevy_ecs/src/archetype.rs

```diff
@@ -395,14 +491,17 @@ impl SparseSetIndex for ArchetypeComponentId {
     }
 }
 
+/// The backing store of all [`Archetype`]s within a [`World`].
```

Comment:
This doc comment would be more helpful with a one sentence summary of what an archetype is.

Reply:
Added a backlink to the module level documentation.